src/StructuredLogger/Strings/Strings.cs

```diff
@@ -78,24 +78,32 @@ private static void InitializeRegex()
 
             TargetDoesNotExistBeforeTargetMessage = CreateRegex(GetString("TargetDoesNotExistBeforeTargetMessage"), 2);
 
-            string copyingFileFrom = GetString("Copy.FileComment");
-            string copyingFileFromEscaped = Escape(copyingFileFrom);
+            string copyingFileFromEscaped = Escape(GetString("Copy.FileComment"));
             CopyingFileFromRegex = new Regex(copyingFileFromEscaped
                 .Replace(@"\{0}", @"(?<From>[^\""]+)")
-                .Replace(@"\{1}", @"(?<To>[^\""]+)")
-                );
+                .Replace(@"\{1}", @"(?<To>[^\""]+)"), RegexOptions.Compiled);
 
             CreatingHardLinkRegex = new Regex(Escape(GetString("Copy.HardLinkComment"))
                 .Replace(@"\{0}", @"(?<From>[^\""]+)")
-                .Replace(@"\{1}", @"(?<To>[^\""]+)")
-                );
+                .Replace(@"\{1}", @"(?<To>[^\""]+)"), RegexOptions.Compiled);

```

Comment:
I noticed these weren't optimized (but run quite a bit on some builds) - happy to separate this optimization into another PR to keep things clean!

Reply:
This is fine, thanks.

I benchmarked the difference a while back and it does have benefits:
https://github.com/KirillOsenkov/Benchmarks/blob/29e6de831b11abd9485b16cc24f3eb310e989ed6/src/Tests/Regex.cs#L8-L29